package.json

```diff
@@ -249,16 +250,7 @@
     "commitMessage": "Deploy Storybook [ci-skip]"
   },
   "browserslist": [
-    "last 4 versions and >0.1%",
-    "Firefox ESR",
-    "not IE 11",
-    "Edge 18",
-    "not dead",
-    "not Samsung > 0",
-    "not op_mini all",
-    "not and_ff > 0",
-    "not and_qq > 0",
-    "not and_uc > 0"
+    "extends browserslist-config-monday"
```

Comment:
this is a breaking change....

Reply:
If I understand correctly - this is the exact same configuration we are using in the Monday production, so it's basically just an alignment

Reply:
@orrgottlieb I agree with @shlomitc
It means this is a breaking change only for consumers that aren't using vibe to build monday apps

Reply:
this is still an open source library, which we have other users... @shlomitc @YossiSaadi 

open source is open source, you should go by the community standards for semver